akka-docs-new/src/main/paradox/guide/actors-intro.md

```diff
@@ -0,0 +1,259 @@
+
+The second key change we need in our model is to reinstate encapsulation. Actors react to messages just like objects 
+"react" to methods invoked on them. The difference is that instead of multiple threads "protruding" into our actor and 
+wreaking havoc to internal state and invariants, actors execute independently from the senders of a message, and they 
+react to incoming messages sequentially, one at a time. There is always at most one message being processed, meaning 
+that invariants can be kept without synchronization. This happens automatically without using locks:
+
+TODO: SERIALIZED-TIMELINE-INVARIANTS
+
+In summary, this is what happens when an actor receives a message. It:
+
+ 1. Adds the message to the end of a queue
+ 2. If the actor was not scheduled for execution, it is marked as ready to execute
+ 3. A (hidden) scheduler entity takes the actor and starts executing it
+ 4. Actor picks the message from the front of the queue
+ 5. Actor modifies internal state, sends messages to other actors
+ 6. Actor is unscheduled
+
+To accomplish this behavior, actors have
+	
+ * A Mailbox (the queue where messages end up)	
+ * A Behavior (the state of the actor, internal variables etc.)	
+ * Messages (pieces of data representing a signal, similar to method calls and their parameters)	
+ * An Execution Environment (the machinery that takes actors that have messages to react to and invokes 
+   their message handling code)	
+ * An Address (more on this later)
+
+Messages are put into Mailboxes of Actors, then the Behavior of the actor describes how the actor responds to 
+messages (like sending more messages and/or changing state). The Execution Environment orchestrates a pool of threads 
+to drive all these actions completely transparently.
+
+This is a very simple model and it solves the issues enumerated previously:	
+ * Encapsulation is preserved by decoupling execution from signaling (method calls transfer execution, 
+   message passing does not).	
+ * There is no need for locks. Modifying the internal state of an actor is only possible via messages, which are 
+   processed one at a time eliminating races when trying to keep invariants.	
+ * There are no	locks used anywhere, and senders are not blocked. Millions of actors can be efficiently scheduled on a 
+   dozen of threads reaching the full potential of modern CPUs. Task delegation is the natural mode of operation 
+   for actors.
+ * State of actors is local and not shared, changes and data is propagated via 	messages, which maps to how modern 
+   memory hierarchy actually works.	In many cases this means transferring over only the cache lines that contain the 
+   data in the message while keeping local state and data cached at the original core. The same model maps exactly 
+   to remote communication where state is kept in the RAM of machines and changes/data is propagated over the network 
+   as packets.
+
+### Actors handle error situations gracefully
```

Comment:
This was a rewrite from the original by Ruth. I don't want to revert back and forth for all rounds of review. I will change only if there is consensus what to write.